### ISSUE TITLE
how can i change the charset of my web to gbk?

### ISSUE THREAD

**Issue by xibeilin** [yiisoft/yii2#9479]
I have changed the configuration file like this:

'charset'  => 'GBK',
'language' => 'zh-CN',

but the error message and gii generated files are garbled。
![image](https://cloud.githubusercontent.com/assets/7733210/9382315/457903fc-4772-11e5-8489-b69bddd372e7.png)

**Comment by SVVladymyr** [comment]
Here it is written about the internationalization:
http://www.yiiframework.com/doc-2.0/guide-tutorial-i18n.html

**Comment by cebe** [comment]
gii generated files are UTF8 by default. Why do you need to use another charset? If your charset is not compatible with utf8 you may need to convert all the files which contain messages in UTF8 to your charset to make it work.

**Comment by xibeilin** [comment]
thank you !
Because other issues left over by history, data in the database is made gbk coding, so I used gii generated files have this problem, but what I need to convert the file it? I found yiisoft / yii2 / message / zh-CN /, but this should be done by an international translation, right? Also, I tried to modify the yiisoft / yii2-gii / views / layout / main.php, but not used, what documents I need to convert it? I was not missing out?
